src/components/MyCalendar.tsx

```diff
@@ -53,18 +91,57 @@ const MyCalendar = () => {
     if (minWidth >= 386) {
       return 'grid-cols-2';
     }
-    return 'grid-cols-1';
+    if (minWidth >= 1) {
+      return 'grid-cols-1';
+    }
+    return 'grid-cols-6';
+  };
+
+  const yearBP1 = () => {
+    const minWidth = Math.min(windowWidth, containerWidth);
+    if (minWidth <= 975) {
+      return 'hidden';
+    }
+    return '';
   };
 
+  const yearBP2 = () => {
+    const minWidth = Math.min(windowWidth, containerWidth);
+    if (minWidth <= 750) {
+      return 'hidden';
+    }
+    return '';
+  };
   return (
     <>
       {/* <div className="container max-w-8xl bg-sky-100"> */}
+      <div className="border border-1">
+        <div className={`${yearBP1} flex items-center justify-center`}>
+          <button
+            className={'bg-white font-bold h-11 hover:bg-gray-200 px-2.5 text-gray-800 text-2xl w-7'}
+            onClick={() => setYear(year - 1)}
+          >
+            {'‹'}
+          </button>
+          <YearEnd className={yearBP2()} onClick={y => setYear(y)} year={year - 2} />
```

Comment:
{showYearEnd() && <YearEnd className={yearBP2()} onClick={y => setYear(y)} year={year - 2} />}

... and so on